Formula/gmp.rb

```diff
@@ -16,12 +16,13 @@ class Gmp < Formula
 
   def install
     ENV.cxx11 if build.cxx11?
-    args = ["--prefix=#{prefix}", "--enable-cxx"]
 
-    # https://github.com/Homebrew/homebrew/issues/20693
-    args << "--disable-assembly" if build.bottle?
+    # XNU release major version
+    osmajor = MacOS.version.to_s[3..-1].to_i + 4
```

Comment:
This is how XNU is versioned
10.6 -> 10
10.7 -> 11
10.8 -> 12
10.9 -> 13
...
10.12 -> 16

There's no reason to shell out if we know the answer.